[2019-12-05 20:24:33,075]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块1条用例，测试数据是{'mobilephone':'13245679319','pwd':'123456'}
[2019-12-05 20:24:33,367]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

[2019-12-05 20:24:33,409]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}，测试结果是：FAILED
[2019-12-05 20:24:33,412]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块2条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:24:33,702]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:24:33,703]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块3条用例，测试数据是{'mobilephone':'83541781424','pwd':'123456','regname':'xiaoming'}
[2019-12-05 20:24:33,885]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:24:33,886]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块4条用例，测试数据是{'mobilephone':'18296208160','pwd':'12345'}
[2019-12-05 20:24:34,087]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}，测试结果是：PASS
[2019-12-05 20:24:34,088]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块5条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:24:34,092]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块6条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:24:34,464]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":null,"msg":"登录成功"}，测试结果是：PASS
[2019-12-05 20:24:34,465]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块7条用例，测试数据是{'mobilephone':'','pwd':'123456'}
[2019-12-05 20:24:34,685]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:24:34,686]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块8条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:24:34,940]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:24:34,940]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块9条用例，测试数据是{'mobilephone':'13541781424','pwd':'1234567'}
[2019-12-05 20:24:35,215]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:24:35,216]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块10条用例，测试数据是{'mobilephone':'18296208190','pwd':'123456'}
[2019-12-05 20:24:35,581]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:24:35,820]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块11条用例，测试数据是{'mobilephone':'13541781424','amount':'200'}
[2019-12-05 20:24:36,372]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'sta[36 chars]id': 1123484, 'regname': '小蜜蜂', 'pwd': 'E10ADC[146 chars]值成功'} != {'sta[36 chars]id': 228735, 'regname': '小蜜蜂', 'pwd': 'E10ADC3[144 chars]值成功'}
Diff is 645 characters long. Set self.maxDiff to None to see it.
[2019-12-05 20:24:36,414]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":{"id":228735,"regname":"小蜜蜂","pwd":"E10ADC3949BA59ABBE56E057F20F883E","mobilephone":"13541781424","leaveamount":"93400.00","type":"1","regtime":"2019-12-03 16:52:06.0"},"msg":"充值成功"}，测试结果是：FAILED
[2019-12-05 20:24:36,415]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块12条用例，测试数据是{'mobilephone':'','amount':'1000'}
[2019-12-05 20:24:36,604]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:24:36,604]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块13条用例，测试数据是{'mobilephone':'13541781424','amount':' '}
[2019-12-05 20:24:36,779]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20118","data":null,"msg":"请输入数字"}，测试结果是：PASS
[2019-12-05 20:24:36,779]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块14条用例，测试数据是{'mobilephone':'23541781424','amount':'10003'}
[2019-12-05 20:24:37,053]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:24:37,054]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块15条用例，测试数据是{'mobilephone':'13541781424','amount':'-1000'}
[2019-12-05 20:24:37,261]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20117","data":null,"msg":"请输入范围在0到50万之间的正数金额"}，测试结果是：PASS
[2019-12-05 20:26:44,258]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块1条用例，测试数据是{'mobilephone':'13245679320','pwd':'123456'}
[2019-12-05 20:26:44,502]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

[2019-12-05 20:26:44,578]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}，测试结果是：FAILED
[2019-12-05 20:26:44,581]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块2条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:26:44,748]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:26:44,749]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块3条用例，测试数据是{'mobilephone':'83541781424','pwd':'123456','regname':'xiaoming'}
[2019-12-05 20:26:44,935]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:26:44,936]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块4条用例，测试数据是{'mobilephone':'18296208160','pwd':'12345'}
[2019-12-05 20:26:45,115]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}，测试结果是：PASS
[2019-12-05 20:26:45,116]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块5条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:26:45,120]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块6条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:26:45,400]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":null,"msg":"登录成功"}，测试结果是：PASS
[2019-12-05 20:26:45,402]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块7条用例，测试数据是{'mobilephone':'','pwd':'123456'}
[2019-12-05 20:26:45,573]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:26:45,574]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块8条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:26:45,770]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:26:45,771]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块9条用例，测试数据是{'mobilephone':'13541781424','pwd':'1234567'}
[2019-12-05 20:26:46,069]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:26:46,070]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块10条用例，测试数据是{'mobilephone':'18296208190','pwd':'123456'}
[2019-12-05 20:26:46,342]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:26:46,547]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块11条用例，测试数据是{'mobilephone':'13541781424','amount':'200'}
[2019-12-05 20:26:47,012]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'sta[36 chars]id': 1123484, 'regname': '小蜜蜂', 'pwd': 'E10ADC[146 chars]值成功'} != {'sta[36 chars]id': 228735, 'regname': '小蜜蜂', 'pwd': 'E10ADC3[144 chars]值成功'}
Diff is 643 characters long. Set self.maxDiff to None to see it.
[2019-12-05 20:26:47,054]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":{"id":228735,"regname":"小蜜蜂","pwd":"E10ADC3949BA59ABBE56E057F20F883E","mobilephone":"13541781424","leaveamount":"93600.00","type":"1","regtime":"2019-12-03 16:52:06.0"},"msg":"充值成功"}，测试结果是：FAILED
[2019-12-05 20:26:47,056]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块12条用例，测试数据是{'mobilephone':'','amount':'1000'}
[2019-12-05 20:26:47,232]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:26:47,234]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块13条用例，测试数据是{'mobilephone':'13541781424','amount':' '}
[2019-12-05 20:26:47,413]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20118","data":null,"msg":"请输入数字"}，测试结果是：PASS
[2019-12-05 20:26:47,414]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块14条用例，测试数据是{'mobilephone':'23541781424','amount':'10003'}
[2019-12-05 20:26:47,587]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:26:47,589]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块15条用例，测试数据是{'mobilephone':'13541781424','amount':'-1000'}
[2019-12-05 20:26:47,758]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20117","data":null,"msg":"请输入范围在0到50万之间的正数金额"}，测试结果是：PASS
[2019-12-05 20:27:00,685]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块1条用例，测试数据是{'mobilephone':'13245679321','pwd':'123456'}
[2019-12-05 20:27:00,933]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

[2019-12-05 20:27:00,977]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}，测试结果是：FAILED
[2019-12-05 20:27:00,979]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块2条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:27:01,178]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:27:01,179]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块3条用例，测试数据是{'mobilephone':'83541781424','pwd':'123456','regname':'xiaoming'}
[2019-12-05 20:27:01,344]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:27:01,345]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块4条用例，测试数据是{'mobilephone':'18296208160','pwd':'12345'}
[2019-12-05 20:27:01,516]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}，测试结果是：PASS
[2019-12-05 20:27:01,516]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块5条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:27:01,520]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块6条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:27:01,802]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":null,"msg":"登录成功"}，测试结果是：PASS
[2019-12-05 20:27:01,803]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块7条用例，测试数据是{'mobilephone':'','pwd':'123456'}
[2019-12-05 20:27:01,983]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:27:01,984]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块8条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:27:02,178]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:27:02,179]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块9条用例，测试数据是{'mobilephone':'13541781424','pwd':'1234567'}
[2019-12-05 20:27:02,485]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:27:02,486]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块10条用例，测试数据是{'mobilephone':'18296208190','pwd':'123456'}
[2019-12-05 20:27:02,797]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:27:03,026]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块11条用例，测试数据是{'mobilephone':'13541781424','amount':'200'}
[2019-12-05 20:27:03,595]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'sta[36 chars]id': 1123484, 'regname': '小蜜蜂', 'pwd': 'E10ADC[146 chars]值成功'} != {'sta[36 chars]id': 228735, 'regname': '小蜜蜂', 'pwd': 'E10ADC3[144 chars]值成功'}
Diff is 645 characters long. Set self.maxDiff to None to see it.
[2019-12-05 20:27:03,637]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":{"id":228735,"regname":"小蜜蜂","pwd":"E10ADC3949BA59ABBE56E057F20F883E","mobilephone":"13541781424","leaveamount":"93800.00","type":"1","regtime":"2019-12-03 16:52:06.0"},"msg":"充值成功"}，测试结果是：FAILED
[2019-12-05 20:27:03,638]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块12条用例，测试数据是{'mobilephone':'','amount':'1000'}
[2019-12-05 20:27:03,857]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:27:03,858]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块13条用例，测试数据是{'mobilephone':'13541781424','amount':' '}
[2019-12-05 20:27:04,042]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20118","data":null,"msg":"请输入数字"}，测试结果是：PASS
[2019-12-05 20:27:04,043]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块14条用例，测试数据是{'mobilephone':'23541781424','amount':'10003'}
[2019-12-05 20:27:04,230]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:27:04,232]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块15条用例，测试数据是{'mobilephone':'13541781424','amount':'-1000'}
[2019-12-05 20:27:04,436]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20117","data":null,"msg":"请输入范围在0到50万之间的正数金额"}，测试结果是：PASS
[2019-12-05 20:30:38,683]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块1条用例，测试数据是{'mobilephone':'13245679322','pwd':'123456'}
[2019-12-05 20:30:38,981]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

[2019-12-05 20:30:39,022]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}，测试结果是：FAILED
[2019-12-05 20:30:39,024]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块2条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:30:39,216]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:30:39,218]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块3条用例，测试数据是{'mobilephone':'83541781424','pwd':'123456','regname':'xiaoming'}
[2019-12-05 20:30:39,397]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:30:39,398]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块4条用例，测试数据是{'mobilephone':'18296208160','pwd':'12345'}
[2019-12-05 20:30:39,599]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}，测试结果是：PASS
[2019-12-05 20:30:39,600]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块5条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:30:39,604]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块6条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:30:39,884]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":null,"msg":"登录成功"}，测试结果是：PASS
[2019-12-05 20:30:39,885]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块7条用例，测试数据是{'mobilephone':'','pwd':'123456'}
[2019-12-05 20:30:40,059]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:30:40,062]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块8条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:30:40,274]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:30:40,275]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块9条用例，测试数据是{'mobilephone':'13541781424','pwd':'1234567'}
[2019-12-05 20:30:40,562]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:30:40,563]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块10条用例，测试数据是{'mobilephone':'18296208190','pwd':'123456'}
[2019-12-05 20:30:40,864]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:30:41,084]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块11条用例，测试数据是{'mobilephone':'13541781424','amount':'200'}
[2019-12-05 20:30:41,602]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'sta[36 chars]id': 1123484, 'regname': '小蜜蜂', 'pwd': 'E10ADC[146 chars]值成功'} != {'sta[36 chars]id': 228735, 'regname': '小蜜蜂', 'pwd': 'E10ADC3[144 chars]值成功'}
Diff is 644 characters long. Set self.maxDiff to None to see it.
[2019-12-05 20:30:41,642]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":{"id":228735,"regname":"小蜜蜂","pwd":"E10ADC3949BA59ABBE56E057F20F883E","mobilephone":"13541781424","leaveamount":"94000.00","type":"1","regtime":"2019-12-03 16:52:06.0"},"msg":"充值成功"}，测试结果是：FAILED
[2019-12-05 20:30:41,643]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块12条用例，测试数据是{'mobilephone':'','amount':'1000'}
[2019-12-05 20:30:41,853]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:30:41,854]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块13条用例，测试数据是{'mobilephone':'13541781424','amount':' '}
[2019-12-05 20:30:42,035]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20118","data":null,"msg":"请输入数字"}，测试结果是：PASS
[2019-12-05 20:30:42,036]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块14条用例，测试数据是{'mobilephone':'23541781424','amount':'10003'}
[2019-12-05 20:30:42,202]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:30:42,204]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块15条用例，测试数据是{'mobilephone':'13541781424','amount':'-1000'}
[2019-12-05 20:30:42,391]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20117","data":null,"msg":"请输入范围在0到50万之间的正数金额"}，测试结果是：PASS
[2019-12-05 20:33:04,195]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块1条用例，测试数据是{'mobilephone':'13245679323','pwd':'123456'}
[2019-12-05 20:33:04,467]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'status': 1, 'code': '10001', 'data': None, 'msg': '注册成功'} != {'status': 0, 'code': '20110', 'data': None, 'msg': '手机号码已被注册'}
- {'code': '10001', 'data': None, 'msg': '注册成功', 'status': 1}
?             ---                           --             ^

+ {'code': '20110', 'data': None, 'msg': '手机号码已被注册', 'status': 0}
?           +++                           ++++++               ^

[2019-12-05 20:33:04,509]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20110","data":null,"msg":"手机号码已被注册"}，测试结果是：FAILED
[2019-12-05 20:33:04,510]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块2条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:33:04,706]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:33:04,707]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块3条用例，测试数据是{'mobilephone':'83541781424','pwd':'123456','regname':'xiaoming'}
[2019-12-05 20:33:04,938]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:33:04,938]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块4条用例，测试数据是{'mobilephone':'18296208160','pwd':'12345'}
[2019-12-05 20:33:05,114]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20108","data":null,"msg":"密码长度必须为6~18"}，测试结果是：PASS
[2019-12-05 20:33:05,114]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行register模块5条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:33:05,118]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块6条用例，测试数据是{'mobilephone':'13541781424','pwd':'123456'}
[2019-12-05 20:33:05,394]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":null,"msg":"登录成功"}，测试结果是：PASS
[2019-12-05 20:33:05,395]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块7条用例，测试数据是{'mobilephone':'','pwd':'123456'}
[2019-12-05 20:33:05,585]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:33:05,586]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块8条用例，测试数据是{'mobilephone':'13541781424','pwd':''}
[2019-12-05 20:33:05,781]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"密码不能为空"}，测试结果是：PASS
[2019-12-05 20:33:05,782]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块9条用例，测试数据是{'mobilephone':'13541781424','pwd':'1234567'}
[2019-12-05 20:33:06,090]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:33:06,091]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行login模块10条用例，测试数据是{'mobilephone':'18296208190','pwd':'123456'}
[2019-12-05 20:33:06,443]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}，测试结果是：PASS
[2019-12-05 20:33:06,641]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块11条用例，测试数据是{'mobilephone':'13541781424','amount':'200'}
[2019-12-05 20:33:07,187]-[ERROR]-[log_module.py-test_log]-[日志信息]:出错啦错误是{'sta[36 chars]id': 1123484, 'regname': '小蜜蜂', 'pwd': 'E10ADC[146 chars]值成功'} != {'sta[36 chars]id': 228735, 'regname': '小蜜蜂', 'pwd': 'E10ADC3[144 chars]值成功'}
Diff is 645 characters long. Set self.maxDiff to None to see it.
[2019-12-05 20:33:07,252]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":1,"code":"10001","data":{"id":228735,"regname":"小蜜蜂","pwd":"E10ADC3949BA59ABBE56E057F20F883E","mobilephone":"13541781424","leaveamount":"94200.00","type":"1","regtime":"2019-12-03 16:52:06.0"},"msg":"充值成功"}，测试结果是：FAILED
[2019-12-05 20:33:07,253]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块12条用例，测试数据是{'mobilephone':'','amount':'1000'}
[2019-12-05 20:33:07,443]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}，测试结果是：PASS
[2019-12-05 20:33:07,444]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块13条用例，测试数据是{'mobilephone':'13541781424','amount':' '}
[2019-12-05 20:33:07,629]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20118","data":null,"msg":"请输入数字"}，测试结果是：PASS
[2019-12-05 20:33:07,629]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块14条用例，测试数据是{'mobilephone':'23541781424','amount':'10003'}
[2019-12-05 20:33:07,826]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20109","data":null,"msg":"手机号码格式不正确"}，测试结果是：PASS
[2019-12-05 20:33:07,826]-[INFO]-[log_module.py-test_log]-[日志信息]:正在执行recharge模块15条用例，测试数据是{'mobilephone':'13541781424','amount':'-1000'}
[2019-12-05 20:33:08,077]-[INFO]-[log_module.py-test_log]-[日志信息]:实际结果是{"status":0,"code":"20117","data":null,"msg":"请输入范围在0到50万之间的正数金额"}，测试结果是：PASS
